codec-http/src/main/java/io/netty/handler/codec/http/multipart/DefaultHttpDataFactory.java

```diff
@@ -183,65 +173,49 @@ private static void checkHttpDataSize(HttpData data) {
 
     @Override
     public Attribute createAttribute(HttpRequest request, String name, String value) {
+        Attribute attribute;
```

Comment:
nit: make this final

Reply:
Can't be done here because its assigned in the exception